components/esp_lvgl_port/include/esp_lvgl_port.h

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
Please consider introducing `esp_lvgl_port_xxx.h` headers. It could make this file a little clearer.
However, initial version can stay like this